docs/user/operators.md

```diff
@@ -8,7 +8,18 @@ In this tutorial we will be using the [Operator Lifecycle Manager (OLM)](https:/
 
 Kubeapps assumes a working Kubernetes cluster (v1.12+) and [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) installed and configured to talk to your Kubernetes cluster. Users following this tutorial require to have admin privileges in the cluster in order to install and manage Operators.
 
-## Step 1: Install the Operator Lifecycle Manager (OLM)
+## Step 1: Enable Operators support in Kubeapps
+
+Since Kubeapps v2.4.3, Operators support is disabled by default.
```

Comment:
I'd also mention the chart version, given it's what users select in Helm when installing.

Reply:
Right, added!